### PR TITLE
Add basic documentation + small fixes

### PR DESCRIPTION
**Pull request by igoumiri**
Add some basic documentation for the API reference. Also clean up some of the imports so that the imported classes don't appear as duplicates in the API reference. I wonder if there is a cleaner way to do that than importing `Thing as _Thing` like I did.

Add a README in `docs/` to explain how to build the docs.

Also fix some leftover `x2y` -> `x_to_y` references to util functions, mostly in tests.

### CODE REVIEW

File: ssapy/body.py [hunk 1]
```diff
@@ -103,7 +107,7 @@ def __init__(self):
         from jplephem.spk import SPK
         from . import datadir
 
-        fn = os.path.join(datadir, "de430.bsp")  # https://naif.jpl.nasa.gov/pub/naif/LUCY/kernels/spk/de430s.bsp.lbl
+        fn = os.path.join(datadir, "de430.bsp")  # https://naif.jpl.nasa.gov/pub/naif/LUCY/kernels/spk/de430s.bsp
```
Comment (aperloff) [inline ssapy/body.py:110]: The `.lbl` part of the URL takes you to the fild documentation, which I think was the point. Please return the url to what it was or list both the source URL and the documentation URL.

File: ssapy/body.py [hunk 2]
```diff
@@ -134,7 +138,7 @@ def __init__(self):
...
```
Comment (aperloff) [inline ssapy/body.py:141]: Same comment as the one above.

File: docs/README.rst [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+.. code-block:: bash
+
+    make html
+
+(run it twice to generate all files.)
+
+Then open `_build/html/index.html` to browse the docs locally.
+
+Note that if you updated docstrings, you'll need to re-build and re-install ssapy before re-building the docs.
```
[the remark this answers is not in the document]
Comment (igoumiri) [reply docs/README.rst:22]: Nice! I tried but I'm using python 3.9 and the version of sphinx-autobuild I have installed requires python 3.10+ (we should pin versions!)

I still added it to the README!

File: docs/source/installation.rst [hunk 4]
```diff
@@ -34,7 +34,7 @@ These requirements can be easily installed on most modern macOS and Linux system
        .. code-block:: console
 
           brew update
-          brew install gcc git git-lfs python3 svn graphviz
+          brew install gcc git git-lfs python3 graphviz
```
[the remark this answers is not in the document]
Comment (igoumiri) [reply docs/source/installation.rst:37]: It couldn't tell why it was needed, I didn't install it and didn't run into any issues, and it didn't seem required for other platforms (lines above) so I assumed it was outdated. Is it still needed?

File: ssapy/body.py (hunk 1 above)
Comment (igoumiri) [reply ssapy/body.py]: Fixed

File: .gitattributes [hunk 5]
```diff
@@ -6,5 +6,7 @@ ssapy/data/de430.bsp filter=lfs diff=lfs merge=lfs -text
 ssapy/data/gggrx_1200a_sha.lbl filter=lfs diff=lfs merge=lfs -text
 ssapy/data/gggrx_1200a_sha.tab filter=lfs diff=lfs merge=lfs -text
 ssapy/data/moon_pa_de440_200625.bpc filter=lfs diff=lfs merge=lfs -text
+*.bsp filter=lfs diff=lfs merge=lfs -text
```
[the remark this answers is not in the document]
Comment (igoumiri) [reply .gitattributes]: Done!

**Pull request merged**